config_test.go

```diff
@@ -25,3 +29,23 @@ func TestLoadBadConfig(t *testing.T) {
 		t.Errorf("\nexpected:\n%v\ngot:\n%v", expected, err.Error())
 	}
 }
+
+func TestPrometheusTimeout(t *testing.T) {
+	initConfig()
```

Comment:
This is pulling the file off the filesystem, it'd be better in the test to pass in the config to be used to probeRequestHandler. It's already setup to allow you to inject a config.